### PR TITLE
Add a setting to change the submit (and new line) key

### LINKED ISSUE #274: Invert Enter key behaviour
Text is being submitted by pressing the `Enter` key and adding a new line by pressing `Shift+Enter`. Some people may want to invert this behaviour.

(Request from https://threema-forum.de/thread-3299.html)

### PR DESCRIPTION
**Pull request by lgrahl**
A little :christmas_tree:  :gift: for all the Threema Web users.

Translations are WIP.

Resolves #274

### CODE REVIEW

File: public/i18n/de.json [hunk 1]
```diff
@@ -307,6 +307,11 @@
             "SHOW_NOTIFICATIONS": "Desktopbenachrichtigungen anzeigen",
             "SHOW_PREVIEW": "Nachrichteninhalt in Benachrichtigungen anzeigen",
             "PLAY_SOUND": "Ton abspielen"
+        },
+        "composearea": {
+            "COMPOSE_AREA": "Textnachrichtverfassungsdingsbumswasweissichdenn",
```
Comment (dbrgn) [inline public/i18n/de.json:312]: "Eingabefeld"?

**Review by dbrgn — CHANGES_REQUESTED** [review]
Looks good in general, I didn't test it though!

Did you try it with at least Firefox and a Chromium based browser?

**Comment by lgrahl** [comment]
> Did you try it with at least Firefox and a Chromium based browser?

Yup, works fine so far.

**Review by dbrgn — APPROVED** [review]
Thanks!

**Comment by rugk** [comment]
> A little :christmas_tree: :gift: for all the Threema Web users.

Hehe, thanks for that! And happy holidays you all! :smiley: :tada:

**Pull request merged**